### PR TITLE
OS X platform name is darwin.

### PR DESCRIPTION
**Pull request by vishrutshah**
Reference: https://nodejs.org/api/process.html#process_process_platform

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: Tools/gulp/gulp-regenerate-expected.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ var PluginError = gutil.PluginError;
 
 var isWindows = (process.platform.lastIndexOf('win') === 0);
 var isLinux= (process.platform.lastIndexOf('linux') === 0);
-var isMac = (process.platform.lastIndexOf('mac') === 0);
+var isMac = (process.platform.lastIndexOf('darwin') === 0);
```
Comment (fearthecowboy) [inline Tools/gulp/gulp-regenerate-expected.js]: can you apply this to the `gulpfile.js` file too?

Comment (vishrutshah) [inline Tools/gulp/gulp-regenerate-expected.js]: Sure. Thanks for pointing it out.

**Comment by fearthecowboy** [comment]
LGTM 👍

**Comment by vishrutshah** [comment]
@fearthecowboy Thanks for the review!!

**Pull request merged**